addons/event/models/event.py

```diff
@@ -120,10 +120,10 @@ def _compute_seats(self):
     date_tz = fields.Selection('_tz_get', string='Timezone', default=lambda self: self.env.user.tz)
     date_begin = fields.Datetime(
         string='Start Date', required=True,
-        readonly=True, states={'draft': [('readonly', False)]})
+        track_visibility=True, states={'done': [('readonly', True)]})
```

Comment:
NB : track_visibility=True will do nothing, you have to choose between 'onchange' or 'always'